include/tiramisu/core.h

```diff
@@ -1069,6 +1099,20 @@ class buffer
      * The location of the buffer (host if in memory, else where on GPU).
      */
     cuda_ast::memory_location location;
+	
+     /**
+     * specify which memory is tagged when using the gpu. By default it is equal to 'g' for global memory, 
+     * it can be set to 'c' to tag constant memory. 
+     * This field is modified when creatin a gpu buffer.
+     */
+     char tag;
```

Comment:
There is already a mechanism for tagging buffers rights? (tag_global ....)
How does that work? Better use the same mechanism instead of creating a new mechanism.